content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -205,6 +205,19 @@ field in the quota spec.
 
 A quota is matched and consumed only if `scopeSelector` in the quota spec selects the pod.
 
+Quota for the following resources can be scoped to Priority class, with the use of `scopeSelector`:
+
+* `limits.cpu`
+* `limits.memory`
+* `limits.ephemeral-storage`
+* `count/pods`
```

Comment:
nit: I'd list this first

Reply:
`count/pods` and `pods` works the same, but for consistency with the previous section, I'd suggest to keep `pods`.